NickvisionMoney.Shared/Controllers/TransferDialogController.cs

```diff
@@ -215,4 +218,56 @@ public TransferCheckStatus UpdateTransfer(string destPath, string? destPassword,
         Transfer.ConversionRate = conversionRate;
         return TransferCheckStatus.Valid;
     }
+
+    private static Dictionary<string, string> tmpCacheSoIDontGetRateLimited = new();
+    
+    public async Task<(decimal, decimal)?> GetConversionRatesAsync()
+    {
+        if(DestinationCurrencyCode is null)
+        {
+            return null;
+        }
+        
+        if(SourceCurrencyCode == DestinationCurrencyCode)
+        {
+            return (1.0m, 1.0m);
+        }
+        
+        var apiUrl = $"https://open.er-api.com/v6/latest/{SourceCurrencyCode}";
+        
+        try
+        {
+            var httpClient = new HttpClient();
```

Comment:
`using var httpClient = new HttpClient();`, then it will be automatically disposed at the end of the block, so you can remove `httpClient.Dispose();`. And move it inside `if` so it will not be created if not needed.